app/src/main/java/com/kickstarter/services/ApiClient.java

```diff
@@ -233,6 +233,23 @@ public ApiClient(final @NonNull ApiService service, final @NonNull Gson gson) {
       .subscribeOn(Schedulers.io());
   }
 
+
+  @Override
+  public @NonNull Observable<SurveyResponse> fetchSurveyResponse(final @NonNull int surveyResponseId) {
```

Comment:
don't need the `@NonNull` annotation for a primitive type. i see it's a vestige of an older commit but does that make sense?